packages/ui/components/Transaction.vue

```diff
@@ -47,7 +55,7 @@ export default {
     },
   },
   methods: {
-    calculateTimeAgo,
+    formatTimestamp,
```

Comment:
We could use a computed property `formattedTimestamp` to avoid adding logic to the template making a function call from there